examples/any-data-type/any_data_type.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina/io;
+
+// This function returns a value of the `anydata` type.
+function getValue() returns (anydata) {
+    string name = "cat";
+    return name;
+}
+
+public function main() {
+    // In this example, the variable named `a` of the `anydata` type holds an `int` value.
+    anydata a = 5;
+    io:println(a);
+
+    // First, the variable of the `anydata` type needs to be cast to the required type (`int` in this example) as shown here.
```

Comment:
How about saying
```suggestion
    // First, the variable of type `anydata` needs to be asserted to be of the required type (`int` in this example) 
```

I think we should be using the word "assert" instead of "cast" now.

Reply:
done